src/main/java/reposense/system/DashboardServer.java

```diff
@@ -0,0 +1,56 @@
+package reposense.system;
+
+import java.awt.Desktop;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.file.Path;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import net.freeutils.httpserver.HTTPServer;
+
+/**
+ * Handles starting the dashboard for the server
```

Comment:
Missing `.` at the end.
`Handles starting of the server to display the dashboard.`